CHANGELOG.md

```diff
@@ -12,10 +12,14 @@ x.x.x Release notes (yyyy-MM-dd)
 * APIs are backwards compatible with all previous releases of Realm JavaScript in the 10.5.x series.
 * File format: generates Realms with format v22 (reads and upgrades file format v5 or later for non-synced Realm, upgrades file format v10 or later for synced Realms).
 
+### Fixed
+* Fixed flexible sync crash when updating subscriptions after token expiry ([#4421](https://github.com/realm/realm-js/issues/4421), since v10.12.0).
```

Comment:
```suggestion
* Fixed flexible sync crash when updating subscriptions after token expiry. ([#4421](https://github.com/realm/realm-js/issues/4421), since v10.12.0)
```